src/target/adiv5.h

```diff
@@ -95,6 +95,14 @@
 #define AP_DESIGNER_TEXAS        0x017
 #define AP_DESIGNER_ATMEL        0x01f
 #define AP_DESIGNER_STM          0x020
+/* CPU2 for STM32W(L|B) uses ARM JEDEC continuation (4) and
+ * not STM ARM JEDEC continuation (0) as for CPU1.
+ * See RM0453:
```

Comment:
what does "RM0453" refer to (could this be made into a link to the appropriate document?)